README.md

```diff
@@ -41,3 +41,112 @@ bazel run -- @buildifier_prebuilt//:buildifier ARGS
 Checkout [the releases
 page](https://github.com/keith/buildifier-prebuilt/releases) for
 snippets for your WORKSPACE
+
+### Specify Version of Buildtools
+
+If you would like to specify a specific version of buildtools to use, you can do one of the
```

Comment:
```suggestion
By default releases of these rules include the most up to date versions of the tools at the current time. If you would like to specify a specific version of buildtools to use, you can do one of the
```

Maybe worth stating to make it super clear that this isn't require unless you feel strongly about the specific version?

Reply:
SGTM. Done.